1_fetch.R

```diff
@@ -115,11 +115,21 @@ p1_targets_list <- list(
     iteration = "group"
   ),
 
-  # Map over groups of sites to download data   
+  # Map over groups of sites to download data.
+  # Note that because error = 'continue', {targets} will attempt to build all 
+  # of the "branches" represented in p1_site_counts_grouped, even if one branch 
+  # returns an error. This way, we will not need to re-build branches that have
+  # already run successfully. However, if a branch fails, {targets} will throw
+  # an error here because it cannot merge the individual branches into a single
+  # data frame. The error(s) associated with the failed branch will therefore 
+  # need to be resolved before the full target can be successfully built. A 
+  # common reason a branch may fail is due to WQP timeout errors. Timeout errors 
+  # can sometimes be resolved by waiting a few hours and retrying tar_make().
```

Comment:
I simulated a fake error to check the messages. Two different things happen, 1) the `built pattern p1_wqp_data_aoi` part that happens after each branch is complete does not happen, and then 2) the pipeline fails on the _next_ target that uses the branched targets result because the branching step was not completed. So, I think we may want to edit this comment to reflect this. I'll make my suggestions on the appropriate line(s).

With a failing branch:

```r
> tar_make()
Linking to GEOS 3.9.1, GDAL 3.2.1, PROJ 7.2.1
v skip target p1_global_grid
v skip target p1_wqp_params_yml
v skip target p1_AOI
v skip target p1_wqp_params
v skip target p1_AOI_sf
v skip target p1_char_names
v skip target p1_global_grid_aoi
v skip target p1_similar_char_names_txt
v skip branch p1_wqp_inventory_5ef390b1
v skip branch p1_wqp_inventory_0fb220f5
v skip branch p1_wqp_inventory_b26714eb
v skip pattern p1_wqp_inventory
v skip target p1_wqp_inventory_aoi
v skip target p1_wqp_inventory_summary_csv
v skip target p1_site_counts
v skip target p1_site_counts_grouped
* start branch p1_wqp_data_aoi_8532d68c
Retrieving WQP data for sites 1:500
* built branch p1_wqp_data_aoi_8532d68c
* start branch p1_wqp_data_aoi_449aba5d
Retrieving WQP data for sites 501:552
x error branch p1_wqp_data_aoi_449aba5d
Error: pretend error

* start target p1_wqp_data_summary_csv
x error target p1_wqp_data_summary_csv
* end pipeline: 42.14 seconds
Error : could not load dependencies of target p1_wqp_data_summary_csv. invalid 'description' argument
In addition: There were 12 warnings (use warnings() to see them)
Error: callr subprocess failed: could not load dependencies of target p1_wqp_data_summary_csv. invalid 'description' argument
Visit https://books.ropensci.org/targets/debugging.html for debugging advice.
Run `rlang::last_error()` to see where the error occurred.
```

No failing branches:

```r
> tar_make()
Linking to GEOS 3.9.1, GDAL 3.2.1, PROJ 7.2.1
v skip target p1_global_grid
v skip target p1_wqp_params_yml
v skip target p1_AOI
v skip target p1_wqp_params
v skip target p1_AOI_sf
v skip target p1_char_names
v skip target p1_global_grid_aoi
v skip target p1_similar_char_names_txt
v skip branch p1_wqp_inventory_5ef390b1
v skip branch p1_wqp_inventory_0fb220f5
v skip branch p1_wqp_inventory_b26714eb
v skip pattern p1_wqp_inventory
v skip target p1_wqp_inventory_aoi
v skip target p1_wqp_inventory_summary_csv
v skip target p1_site_counts
v skip target p1_site_counts_grouped
* start branch p1_wqp_data_aoi_8532d68c
Retrieving WQP data for sites 1:500
* built branch p1_wqp_data_aoi_8532d68c
* start branch p1_wqp_data_aoi_449aba5d
Retrieving WQP data for sites 501:552
* built branch p1_wqp_data_aoi_449aba5d
* built pattern p1_wqp_data_aoi
* start target p1_wqp_data_summary_csv
All good! The expected number of records from the WQP inventory matches the data pull for all characteristics.
* built target p1_wqp_data_summary_csv
* end pipeline: 37.98 seconds
```